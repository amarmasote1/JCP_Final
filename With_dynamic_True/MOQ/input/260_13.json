[
  
  {
    "storeID": "2370",
    "itemID": "29612343103",
    "uom": "EACH",
    "supplyData": {
      "onhandsupply":10
    },
    "demandData": {
        "reserveddemand": 0
      },
    "timestamp": "<$nextMinute>"
    
  }
]