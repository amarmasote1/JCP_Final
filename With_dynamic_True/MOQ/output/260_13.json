{
    "message": "Success",
    "body": [
        {
            "itemID": "29612343103",
            "uom": "EACH",
            "demandData": {
                "reserveddemand": 0
            },
            "markDown": {
                "active": false
            },
            "storeID": "2370",
            "supplyData": {
                "onhandsupply": 10
            },
            "timestamp": "2020-02-28 12:46:54.0000 IST"
        }
    ],
    "status": 200
}