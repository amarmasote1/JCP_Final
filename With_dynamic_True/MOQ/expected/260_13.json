{
    "message": "Success",
    "body": [
        {
            "itemID": "29612343103",
            "uom": "EACH",
            "demandData": {
                "reserveddemand": 0
            },
            "markDown": {
                "active": false
            },
            "storeID": "2370",
            "supplyData": {
                "onhandsupply": 10
            }
        }
    ],
    "status": 200
}